src/main/java/carpetextra/CarpetExtraSettings.java

```diff
@@ -252,6 +253,22 @@ public Integer validate(ServerCommandSource source, ParsedRule<Integer> currentR
             category = {EXTRA, EXPERIMENTAL, DISPENSER}
     )
     public static boolean dragonsBreathConvertsCobbleToEndstone = false;
+
+    @Rule(
+            desc = "Maximum water sucking for sponge",
+            options = {"64"},
+            category = {FEATURE, EXTRA},
+            strict = false
+    )
+    public static int maxSpongeSuck = 64;
+
+    @Rule(
+            desc = "Maximum offset limit for sponge",
+            options = {"7"},
+            category = {FEATURE, EXTRA},
+            strict = false
+    )
+    public static int maxSpongeOffset = 7;
```

Comment:
why not call this maxSpongeDistance?

Reply:
yeah or maxSpongeRange, which one is better?